index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <style>
+    @media (max-width: "700px"){
+  #projects {
+    width: 25%;
+  }
+}
+
+#welcome-section {
+  height: 100vh;
+}
+
+#navbar {
+  position: fixed;
+  top: 0;
+  right: 0;
+}
+</style>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <section id="welcome-section">
+    <h1>These are some of my projects</h1>
+  </section>
```

Comment:
Put the h1 in header tag instead of a section in this part of code

Reply:
@Alibaba2023 👋 

- [ ] Kindly phrase the comments politely as suggestions instead of orders.
  - For example:
    - This is a polite suggestion: "I think it would be better to use `h1` in the header".
 
- [ ] Kindly use reasoning `why` students need to consider your suggestion. Why do students need to use  `h1`  tag? Use reasoning here and you can add any referral link from where students can get hints.